mne/viz/_brain/_brain.py

```diff
@@ -854,6 +854,16 @@ def set_time_interpolation(self, interpolation):
                         assume_sorted=True)
             self._time_interp_inv = _safe_interp1d(idx, self._times)
 
+    def _interpolate_data(self, array, time_idx):
+        from scipy.interpolate import interp1d
+        times = np.arange(self._n_times)
+        act_data = interp1d(
+            times, array, self.time_interpolation, axis=1,
+            assume_sorted=True)(time_idx)
+        ifunc = interp1d(times, self._data['time'])
+        act_time = ifunc(time_idx)
+        return act_data, act_time
```

Comment:
@GuillaumeFavelier this merge commit that you made undid the nice refactoring that made use of `_time_interp_funcs`, we had a single path for interpolation in place and this undid it. It's why CircleCI is headed for failure on `master`:

https://circleci.com/gh/mne-tools/mne-python/18662

Can you look to see if this can be removed?

Reply:
I'm on it

Reply:
My bad for the regression

Reply:
Wait are you sure though? because I added _interpolate_data afterwards.

I fixed it in: https://github.com/mne-tools/mne-python/pull/7380/commits/82b016a933bcc4e155d25bc9a9f04e807f8c675d

Reply:
I opened https://github.com/mne-tools/mne-python/pull/7462